w3d_translator/classes/Color.py

```diff
@@ -0,0 +1,12 @@
+import xml.etree.ElementTree as ET
+from utils import str_to_rgba
+
+
+class Color:
+    def __init__(self, xml: ET.Element):
+        self.xml = xml
+        self.rgba = str_to_rgba(xml.attrib["color"])
+        self.r = self.rgba["r"]
+        self.g = self.rgba["g"]
+        self.b = self.rgba["b"]
+        self.a = self.rgba["a"]
```

Comment:
Not to be "that person" but why are you using a class here? Do you plan on referencing the XML element or the `rgba` attribute again? Do you plan on adding methods to this class? Are you relying on the `Color` type for anything?

If you need the types, have you considered `dataclasses`? Or maybe a straight dict?

Reply:
They'll have a `to_yaml()` method but that's probably it. I'll definitely look into `dataclasses`! Because, yeah, I'm mostly just looking to maintain the OOP feel of the XML data but individual classes does seem overkill

Reply:
Now using the `dataclasses` to type the XML objects in python